tslint.json

```diff
@@ -13,7 +13,6 @@
     "no-var-keyword": true,
     "no-empty": true,
     "no-unused-expression": true,
-    "no-use-before-declare": true,
```

Comment:
This was a breaking change in TSLint 5.0. See [the release notes](https://github.com/palantir/tslint/releases/tag/5.0.0) or https://github.com/palantir/tslint/pull/2235 for more details. We good to remove this?